### DISCUSSION TITLE
Voting Algorithm

### DISCUSSION THREAD

**Discussion by wipfli** [maplibre/maplibre#90] [Governing Board Election]
@nyurik brought the point up in the Slack channel that opavote.com has a bunch of different ranked voting algorithms to choose from. Which one should we take?

https://www.opavote.com/methods/single-transferable-vote

**Comment by lseelenbinder** [comment]
I don't have a strong opinion. Probably Scottish or Meek, since they're recommended.

**Reply by nyurik** [reply]
I agree - Meek seems like the best (more math is good!) approach :)

In depth info: https://blog.opavote.com/2017/04/meek-stv-explained.html
